File: wallendar-app/app/src/main/java/com/td/wallendar/addcharge/ui/AddChargeView.java
package com.td.wallendar.addcharge.ui;

import com.td.wallendar.models.Group;

import java.util.List;

public interface AddChargeView {

    void onGroupsLoadOk(List<Group> groups);

    void onGroupsLoadError();

    void chargeAddedOk();

    void chargeError();

    void setSelectedGroup(final Long groupId);
}

File: wallendar-app/app/src/main/java/com/td/wallendar/addcharge/ui/AddChargeView.kt
package com.td.wallendar.addcharge.ui

import com.td.wallendar.models.Charge
import com.td.wallendar.models.Group

interface AddChargeView {
    fun onGroupsLoadOk(groups: MutableList<Group>)
    fun onGroupsLoadError()
    fun chargeAddedOk(charge: Charge)
    fun chargeError()
    fun setSelectedGroup(groupId: Long)
}